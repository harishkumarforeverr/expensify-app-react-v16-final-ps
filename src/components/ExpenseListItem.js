import React from 'react'; 
import moment from "moment";
import {Link} from "react-router-dom";
import formatAmount from 'indian-currency-formatter';

const ExpenseListItem=({description,amount,createdAt,id })=>{
    return (
        <div> 
           <Link to={`edit/${id}`}>
           <h3>{description} </h3> 
           <p>  
              &#x20B9;{formatAmount(parseFloat(amount,10)) } 
              - 
              {moment(createdAt).format("Do MMM, YYYY")}  
           </p> 
           </Link> 
        </div>
    )
} 

export default ExpenseListItem;