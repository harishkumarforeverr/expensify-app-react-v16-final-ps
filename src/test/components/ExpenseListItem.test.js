import {shallow} from 'enzyme';
import React from 'react';
import expenses from '../fixtures/expenses';
import ExpenseListItem from "../../components/ExpenseListItem";

test('import ExpenseListItem from "../../components/ExpenseListItem"',()=>{
    const wrapper= shallow(<ExpenseListItem {...expenses[2]}  />);
    expect(wrapper).toMatchSnapshot();
})