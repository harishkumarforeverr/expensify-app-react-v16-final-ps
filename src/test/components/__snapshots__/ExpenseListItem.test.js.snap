// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`import ExpenseListItem from "../../components/ExpenseListItem" 1`] = `
<div>
  <Link
    to="edit/2"
  >
    <h3>
      credit Card
       
    </h3>
    <p>
      â‚¹
      4,500
      -
      5th Jan, 1970
    </p>
  </Link>
</div>
`;
